src/SourceCompile/SV3_1aPpTreeShapeListener.cpp

```diff
@@ -556,25 +580,28 @@ void SV3_1aPpTreeShapeListener::enterMacroInstanceNoArgs(
     MacroInfo *macroInf = m_pp->getMacro(macroName);
     if (macroInf) {
       if (macroInf->m_type == MacroInfo::WITH_ARGS) {
-        Location loc(m_pp->getFileId(lineCol.first),
-                     m_pp->getLineNb(lineCol.first), 0,
+        Location loc(m_pp->getFileId(startLineCol.first),
+                     m_pp->getLineNb(startLineCol.first), startLineCol.second,
                      getSymbolTable()->getId(macroName));
-        Location extraLoc(macroInf->m_file, macroInf->m_line, 0);
+        Location extraLoc(macroInf->m_file, macroInf->m_startLine,
+                          macroInf->m_startColumn);
         logError(ErrorDefinition::PP_MACRO_PARENTHESIS_NEEDED, loc, extraLoc);
       }
 
-      IncludeFileInfo info(macroInf->m_line, macroInf->m_file,
-                           m_pp->getSumLineCount() + 1, 0, 0, 0,
-                           IncludeFileInfo::PUSH);
-      m_pp->getSourceFile()->getIncludeFileInfo().push_back(info);
+      m_pp->getSourceFile()->getIncludeFileInfo().emplace_back(
+          macroInf->m_startLine, macroInf->m_file, m_pp->getSumLineCount() + 1,
+          macroInf->m_startColumn,
+          m_pp->getSumLineCount() + 1 +
```

Comment:
I can't do the mental math, but that might be where there is an offset created?

Reply:
It takes the column information directly from value returned from `ParseUtils::getLineColumn` and `ParseUtils::getEndLineColumn`. Only line information is computed and that's coming out correct.

Reply:
This seems like a bug in ParseUtils logic. The two overloads for `getLineColumn` and `getEndLineColumn` aren't doing the same thing. Once I match the logic in both implementations, the result comes out of correct.